linux_os/guide/system/permissions/partitions/mount_option_tmp_nodev/rule.yml

```diff
@@ -47,4 +47,5 @@ template:
         type@rhel7: tmpfs
         type@rhel8: tmpfs
     backends:
-        anaconda: 'off'
+        anaconda@rhel7: 'off'
+        anaconda@rhel8: 'off'
```

Comment:
Fedora and Oracle also had anaconda remediation for `/dev/shm` mount points disabled.

> To get the same result using both old and new templating system,
we need to change the backends list.

But why do we want same results from old and new templating system?

In this case I think the content is better with anaconda disabled for all products.

The reason this is disabled is because `/dev/shm`doesn't exist at the moment of installation, they are managed by [systemd](https://github.com/lnykryn/systemd-rhel/blob/747f3af70c0ee9089e520ccd8917ded397a5cd2a/src/core/mount-setup.c#L91) and created during first boot.
So any product using systemd should have anaconda remediation disabled.

Reply:
I couldn't find any resource documenting management of `/dev/shm` by systemd.

Reply:
Content equivalence between the old and new systems makes it easy to evaluate the new system's state.

Reply:
> Content equivalence between the old and new systems makes it easy to evaluate the new system's state.

I understand this reasoning, but I disagree with hindering generation of content with new templating system.

How about upping the CSV so that it generates the same content as the new system would?